Covid/Quarantine/AddressConfirmationViewController.swift

```diff
@@ -50,29 +50,7 @@ final class AddressConfirmationViewController: UIViewController {
         Defaults.quarantineLatitude = location?.latitude
         Defaults.quarantineLongitude = location?.longitude
 
-        if Defaults.phoneNumber != nil, let token = Defaults.mfaToken {
-            networkService.requestQuarantine(quarantineRequestData: QuarantineRequestData(mfaToken: token)) { [weak self] (result) in
```

Comment:
som slepy, alebo to nikde nemame?

Reply:
`MainViewController. registerForQuarantine`

vynal som ten call z oboch VC's a teraz sa to vola raz po dokonceni celeho flowu